carma-messenger-ui/website/scripts/carma.widgetfw.js

```diff
@@ -34,12 +31,9 @@ CarmaJS.WidgetFramework = (function () {
         };
 
         /*
-            loads the widgets onto the Driver View.Part of implementation truck inspection plugin
```

Comment:
It looks like both the `loadWidgets` and `closeWidgets` functions are solely for the truck inspection plugin, so I think we can remove both of them entirely.

Reply:
I thought so too but when removing these functions the UI doesn't load the event management or emergency response panels either, so I think these need to stay.

Reply:
I realized that I forgot to remove other references to these when testing so I went back and everything works with all references to `loadWidgets` and `closeWidgets` removed. Just fixed this in the latest commit